Xamarin.Forms.Platform.iOS/Renderers/ButtonRenderer.cs

```diff
@@ -170,9 +166,12 @@ protected override void SetBackground(Brush brush)
 					var backgroundImage = this.GetBackgroundImage(brush);
 					backgroundColor = backgroundImage != null ? UIColor.FromPatternImage(backgroundImage) : UIColor.Clear;
 				}
+
+				_useBackgroundBrush = true;
 			}
 
-			Control.BackgroundColor = backgroundColor;
+			if (_useBackgroundBrush)
```

Comment:
Should this be using a locally stored _defaultBackgroundColor instead of a bool check?

Basically how SetBackgroundColor is setup?

```C#
		protected override void SetBackgroundColor(Color color)
		{
			if (IsElementOrControlEmpty)
				return;
#if __MOBILE__
			if (color == Color.Default)
				Control.BackgroundColor = _defaultColor;
			else
				Control.BackgroundColor = color.ToUIColor();
#else
			Control.Layer.BackgroundColor = color == Color.Default ? _defaultColor : color.ToCGColor();
#endif
		}
```